### DISCUSSION TITLE
Erro ao atualizar a versão

### DISCUSSION THREAD

**Discussion by vipeol** [HashLoad/horse#349] [Q&A] [answered]
Olá,

 Ao executar o comando boss update, e atualizar os fontes do projeto, ao tentar compilar esta dando o seguinte erro no arquivo

[dcc32 Error] Horse.ServerStatic.pas(50): E2003 Undeclared identifier: 'TKind'   e logo abaixo outro erro
[dcc32 Error] Horse.ServerStatic.pas(103): E2003 Undeclared identifier: 'Default'

**Comment by viniciussanchez** [comment, marked as answer]
Bom dia, esse problema não é do Horse e sim de algum middleware que você está utilizando...

**Reply by vipeol** [reply]
Obrigado!
